### ISSUE TITLE
Modules containing modules with identical names

### ISSUE THREAD

**Issue by gdkrmr** [julia-vscode/StaticLint.jl#52]
The inner `MyPkg` module should get flagged, `using MyPkg` fails with:

`ERROR: LoadError: cannot replace module MyPkg during compilation`

```julia
module MyPkg
g(x) = 2 + x
module MyPkg
g(x) = 100000000 * x
end
end
```
